digits/dataset/images/classification/test_views.py

```diff
@@ -66,7 +66,7 @@ class BaseViewsTestWithImageset(BaseViewsTest):
     IMAGE_WIDTH     = 10
     IMAGE_CHANNELS  = 3
     BACKEND         = 'lmdb'
-    COMPRESSION     = 'none'
+    COMPRESSION     = 'none' 
```

Comment:
I think trailing spaces should be avoided if possible (especially here since there is no functional change to this line)